pyexchange/fix.py

```diff
@@ -288,7 +288,7 @@ async def _heartbeat(self):
                 m = self.create_message(simplefix.MSGTYPE_HEARTBEAT)
                 await self._write_message(m)
             except ConnectionError as ex:
-                logging.warning(f"Unable to send heartbeat: {ex}")
+                logging.info(f"Unable to send heartbeat: {ex}")
```

Comment:
This should remain a `warning` because it's unexpected behavior.

Reply:
True; was being overly cautious to ensure it logs out but I can change this back